### ISSUE TITLE
Is this project still active - any help needed?

### ISSUE THREAD

**Issue by shymega** [Netizine/StarlingBankClient#1]
Hey.

I'm working on a C# SDK for Starling Bank too, but I don't want to duplicate efforts - perhaps it'd be best to merge the two projects into one? Mine is only barebones right now. Is your wrapper still 'alive'? Do you need any help?

I intend to use the C# SDK/wrapper for my PowerShell module I'm writing in C#.

What do you think?

Thanks!

**Comment by Netizine** [comment]
As far as i know the rest SDK still works and makes integration a lot cleaner. Happy to collaborate.

**Comment by shymega** [comment]
Yeah, I've been testing it for my PowerShell module. The organisation is superb 👌🏻! I've accepted your invite - one thing that Starling has changed at the moment is that webhooks are now done differently -- did/do you intend to support *receiving* webhook payloads?

Thanks.

**Comment by Netizine** [comment]
Sure ill add it this weekend maybe. Otherwise feel fee to add and submit a change request

**Comment by shymega** [comment]
(Updated title and request for clarity)

**Comment by shymega** [comment]
> Sure ill add it this weekend maybe. Otherwise feel fee to add and submit a change request

OK, thanks for letting me know 👍🏻
